lib/enigma.rb

```diff
@@ -0,0 +1,76 @@
+class Enigma
+
+  attr_reader :alphabet
+  def initialize
+    @alphabet = ("a".."z").to_a << " "
+  end
+
+  def todays_date
+    Date.today.strftime(("%d%m%y"))
```

Comment:
I made date a method because I was having trouble using stubs when it was an instance variable. I may try to switch back to an instance variable if I can figure out how to test.